lib/config/presets/internal/helpers.ts

```diff
@@ -37,7 +37,7 @@ export const presets: Record<string, Preset> = {
         extends: ['helpers:pinGitHubActionDigests'],
         extractVersion: '^(?<version>v\\d+\\.\\d+\\.\\d+)$',
         versioning:
-          'regex:^v(?<major>\\d+)(\\.(?<minor>\\d+)\\.(?<patch>\\d+))?$',
+          'regex:^v?(?<major>\\d+)(\\.(?<minor>\\d+)\\.(?<patch>\\d+))?',
```

Comment:
Is dropping the $ necessary?

Reply:
I don't think so

Reply:
```suggestion
          'regex:^v?(?<major>\\d+)(\\.(?<minor>\\d+)\\.(?<patch>\\d+))?$',
```

Reply:
`$` re-added. I just added it since it was a part of your suggestion in the linked discussion.